doc/source/user_guide/basics.rst

```diff
@@ -2284,6 +2284,7 @@ useful if you are reading in data which is mostly of the desired dtype (e.g. num
 non-conforming elements intermixed that you want to represent as missing:
 
 .. ipython:: python
+   :okwarning:
```

Comment:
what warning r u suppressing?

Reply:
trying to parse the string `'apple'`, for which a datetime format can't be guessed. It doesn't error because of `errors='coerce'`

Reply:
great thanks